src/cats/labs/channel.cljc

```diff
@@ -78,64 +64,47 @@
 
     p/Functor
     (-fmap [_ f mv]
-      (let [c (a/chan 1 (map f))]
-        (a/pipe mv c)
-        c))
+      (a/pipe mv (a/chan 1 (map f))))
 
     p/Semigroup
     (-mappend [_ sv sv']
-      (chain-chans sv sv'))
+      (a/merge [sv sv']))
```

Comment:
As far as I understand, merge is not sequential, so it does not a proper impl for mappend. I'm missing something?